platform/linux-generic/m4/odp_libconfig.m4

```diff
@@ -15,4 +15,14 @@ AC_SUBST(_ODP_CONFIG_VERSION_MAJOR)
 _ODP_CONFIG_VERSION_MINOR=_odp_config_version_minor
 AC_SUBST(_ODP_CONFIG_VERSION_MINOR)
 
-ODP_LIBCONFIG([$with_platform])
+##########################################################################
+# Set optional path for the default configuration file
+##########################################################################
+default_config_path="${srcdir}/config/odp-$with_platform.conf"
+
+AC_ARG_WITH([config],
+AS_HELP_STRING([--with-config=FILE      path to the default configuration file],
```

Comment:
I'd prefer to name this --with-config-file. It would be more clear what the option controls and would match ODP_CONFIG_FILE env variable naming.

Reply:
Makes sense. Will fix in V2.